charts/app-config-frontend/templates/deployment.yaml

```diff
@@ -88,6 +88,19 @@ spec:
             successThreshold: {{ .Values.readinessProbe.successThreshold }}
             failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
           {{- end }}
+          {{- if .Values.customStartupProbe }}
```

Comment:
The default value is currently set to `{}`, which I guess will give a TRUE here in this check and the startupProbe will be always `{}`, if the custom one is not changes to something else.

Reply:
Hi @ewelinagr ! No, an empty _collection_ will return `false` (see [docs](https://helm.sh/docs/chart_template_guide/control_structures/#ifelse)). Excerpt:

```
A pipeline is evaluated as false if the value is:
   ...
- an empty collection (map, slice, tuple, dict, array)
```